src/common/store/pages/audio-rewards/slice.ts

```diff
@@ -123,6 +131,25 @@ const slice = createSlice({
       state,
       action: PayloadAction<{ token: string }>
     ) => {},
+    claimChallengeReward: (
+      state,
+      _: PayloadAction<{
```

Comment:
might still be worth giving a name for ts-docs? working to get an eslint setup to prevent unused args, with a pattern of putting `_` in front of the variable, so maybe we do `_action` here?